container/templates/build-docker-compose.j2.yml

```diff
@@ -21,6 +21,8 @@ ansible-container:
     - {{ base_path }}:/ansible-container/{% if params.selinux %}:Z{% endif %}
     {% if params.with_volumes %}{% for vol in params.with_volumes %}
     - {{ vol }}{% endfor %}{% endif %}
-    {% if params.roles_path %}- {{ params.roles_path }}:/local-roles{% endif %}
+    {% for role_path in params.roles_path %}
+    - {{ role_path }}:/local-roles/{{ role_path }}
+    {% endfor %}
   working_dir: /ansible-container/ansible/
```

Comment:
We don't actually use this template anymore. I guess it's OK to update it, but we really should just get rid of it.